src/main/java/org/spdx/sbom/gradle/utils/SpdxDocumentBuilder.java

```diff
@@ -336,6 +336,8 @@ private Optional<SpdxPackage> createMavenModulePackage(
               null);
       spdxPkgBuilder.setExternalRefs(Collections.singletonList(externalRef));
 
+      spdxPkgBuilder.setVersionInfo(moduleId.getVersion());
```

Comment:
is this guaranteed to exist in a valid pom? Do we need to null/empty check it?

Reply:
I imagine it is, but I'll see if there's any kind of spec for that. 

It's working as is on my project with 350 Maven dependencies with no issues.

Reply:
[Looks like version is required](https://maven.apache.org/pom.html#maven-coordinates)

> groupId:artifactId:version are all required fields (although, groupId and version do not need to be explicitly defined if they are inherited from a parent - more on inheritance later).

If the version is provided by the parent then the effective POM should have it